velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -1365,6 +1365,19 @@ void SubstraitVeloxPlanConverter::setSubfieldFilter(
   if (colFilters.size() == 1) {
     filters[common::Subfield(inputName)] = std::move(colFilters[0]);
   } else if (colFilters.size() > 1) {
+    // BigintMultiRange should have been sorted
+    if (colFilters[0]->kind() ==
+        facebook::velox::common::FilterKind::kBigintRange) {
```

Comment:
LGTM. Some minor change: 
`common::FilterKind` can already work I guess.

Reply:
Yes, I changed and verified locally